tests/testament/categories.nim

```diff
@@ -522,6 +522,8 @@ proc processCategory(r: var TResults, cat: Category, options: string) =
   of "untestable":
     # We can't test it because it depends on a third party.
     discard # TODO: Move untestable tests to someplace else, i.e. nimble repo.
-  else:
+  elif os.existsDir("tests" & DirSep &.? cat.string):
```

Comment:
Instead of this ``elif``, keep the ``else`` but count the number of processed tests in the ``for`` loop and if zero, produce the warning/error.